ambry-store/src/main/java/com/github/ambry/store/RemoteTokenTracker.java

```diff
@@ -65,6 +65,10 @@ public class RemoteTokenTracker implements Closeable {
 
   public RemoteTokenTracker(ReplicaId localReplica, ScheduledExecutorService scheduler,
       StoreKeyFactory storeKeyFactory) {
+    if (scheduler == null || localReplica == null) {
+      // TODO [TOMBSTONE]: throw exception
+      logger.error("localReplica and schedule CANNOT BE NULL.");
```

Comment:
can we just throw an exception here.

Reply:
I was thinking,
1. we start with error message and deploy it first.
2. After we monitor the cluster and there is no surprise, the following PR will always throw exception.

Reply:
I don't think that's necessary, we have staging and canary.

Reply:
Sure. Will change it.

Reply:
Changed. Thanks,